test/integ_test/evaluator/azureml_eval/utils.py

```diff
@@ -128,7 +125,7 @@ def get_aml_target():
     )
 
 
-def generate_olive_workspace_config(workspace_config_path):
+def get_olive_workspace_config():
     subscription_id = os.environ.get("WORKSPACE_SUBSCRIPTION_ID")
     if subscription_id is None:
         raise Exception("Please set the environment variable WORKSPACE_SUBSCRIPTION_ID")
```

Comment:
not raise Exception

Reply:
Will address in the future when we refactor the other exceptions in Olive.